source/crud/query/specify-documents-to-return.txt

```diff
@@ -23,6 +23,303 @@ Overview
+document. ``find()`` returns the following ordering of documents when this sort is used on
+the documents matching the query:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 1, "title": "The Brothers Karamazov", "author": "Dostoyevsky", "length": 824 }
+   { "_id": 5, "title": "Cryptonomicon", "author": "Stephenson", "length": 918 }
+   { "_id": 3, "title": "Atlas Shrugged", "author": "Rand", "length": 1088 }
+   { "_id": 6, "title": "A Dance with Dragons", "author": "Martin", "length": 1104 }
+   { "_id": 4, "title": "Infinite Jest", "author": "Wallace", "length": 1104 }
+   { "_id": 2, "title": "Les Misérables", "author": "Hugo", "length": 1462 }
+
+.. _node-fundamentals-skip:
+
+Skip
+----
+
+Use the ``skip()`` method to omit documents from the beginning of the list of
+returned documents for a read operation. You can combine ``skip()`` with
+:ref:`sort() <node-fundamentals-sort>` to omit the top
+(for descending order) or bottom (for ascending order) results for a
+given query. Since the :manual:`order of documents returned
+</reference/method/cursor.sort/#result-ordering>` is not guaranteed in
+the absence of a sort, using ``skip()`` without using ``sort()`` omits
+arbitrary documents.
+
+If the value of ``skip()`` exceeds the number of matched documents for
+a query, then that query returns no documents.
+
+The following example queries the collection for the books with the second and third
+highest lengths. 
+
+- The query matches all documents because the query filter is empty.
+- The descending ``sort()`` method on the ``length`` field returns longer books before shorter books.
+- The ``skip()`` method specifies that the first matching document is omitted from the result.
+
+.. code-block:: javascript
+
+   // define an empty query document
+   const query = {};
+   const sort = { length: -1 };
+   const skip = 1;
+   const cursor = myColl.find(query).sort(sort).skip(1);
+   for await (const doc of cursor) {
+     console.dir(doc);
+   }
+
+Since the query skips the first matching document, the preceding code snippet prints
+the second and third highest length documents:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 4, "name": "Infinite Jest", "author": "Wallace", "length": 1104 },
+   { "_id": 6, "name": "A Dance With Dragons", "author": "Martin", "length": 1104 }
+
+You can also apply ``skip()`` and ``sort()`` by specifying them in an
+``options`` object in your call to the ``find()`` method. The following two
+calls are equivalent:
+
+.. code-block:: javascript
+
+   myColl.find(query).sort({ length: -1 }).skip(1);
+   myColl.find(query, { sort: { length: -1 }, skip: 1});
+
+For more information on the ``options`` settings for the ``find()``
+method, see the
+`API documentation on find() <{+api+}/classes/Collection.html#find>`__.
+
+
+Combine Limit, Sort, and Skip
+-----------------------------
+
+You can combine the ``limit()``, ``sort()``, and ``skip()`` options in a single
```

Comment:
S: since you're talking about the options here, not the methods, you can remove the ()
```suggestion
You can combine the ``limit``, ``sort``, and ``skip`` options in a single
```